docs/quickstart/aws.md

```diff
@@ -0,0 +1,164 @@
+```
+
+Congrats you’ve SSH’d into the server.
+
+In this instance, Coder can be accessed at the url https://fccad1b6c901511b30cf2cf4fbd0973e.pit-1.try.coder.app.
+
+Copy the URL and run the following command to create the workspace owner:
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts - be sure to save use email and password - these are your admin username and password.
+
+You can now access Coder on your local machine with the IP address from your security group by navigating to the `***.try.coder.app` URL and logging in with the username and password.
+
+## Install Coder
+
+For this instance, we will run Coder as a service command, however you can run Coder a multitude of different ways - you can learn more about those here.
+
+In the EC2 instance, run the following command to install Coder
+
+```sh
+curl -fsSL <https://coder.com/install.sh> | sh
+```
+
+## Run Coder
+
+First, edit the `coder.env` file to enable `CODER_TUNNEL` by setting the value to true with the following command:
+
+```sh
+sudo vim /etc/coder.d/coder.env
+```
+<img src="../images/quickstart/aws/aws7.png">
+
+Exit vim and run the following command to start Coder as a system level service:
+
+```sh
+sudo systemctl enable --now coder
+```
+
+The following command will get you information about the Coder launch process
+
+```sh
+journalctl -u coder.service -b
+```
+
+This will return a series of Coder logs, however, embedded in the launch is the URL for accessing Coder.
+
+<img src="../images/quickstart/aws/aws8.png">
+
+In this instance, Coder can be accessed at the url `https://fccad1b6c901511b30cf2cf4fbd0973e.pit-1.try.coder.app`.
+
+Copy the URL and run the following command to create the first user, either on your local machine or in the AWS EC2 instance terminal. 
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts - be sure to save use email and password - these are your admin username and password.
+
+You can now access Coder on your local machine with the relevant  `***.try.coder.app` URL and logging in with the username and password.
+
+## Creating and Uploading Your First Template
+
+First, run `coder template init` to create your first template. You’ll be given a list of possible templates to use. This tutorial will show you how to set up your Coder instance to create a linux based machine on AWS. 
+
+<img src="../images/quickstart/aws/aws9.png">
+
+Press `enter` to select `Develop in Linux` on AWS template. This will return the following: 
+
+<img src="../images/quickstart/aws/aws10.png">
+
+To get started using the AWS template, install the AWS CLI and put the credentials in this account. Follow these instructions to install the AWS CLI and add your credentials.
+
+Since we're running Coder as a system level process, a system user `coder` is created in order to run Coder services. The `coder` user will require access to the AWS credentials to initialize the template and provision workspaces.
```

Comment:
```
Since we're running Coder as a system level process, a system user `coder` is created in order to run Coder services.
```
This line doesn't make sense to me because running Coder as a systemd service doesn't require running as an unprivileged user, that was a decision we made.

Reply:
never said it was unpriv however it does run as a different user?

Reply:
I'm using "unpriv" to refer to the non-root user. I think my confused interpretation stems from the use of the "system level process" language, when it's the systemd unit creating the user and not the `coder server`. I believe you've rephrased a lot of the doc so I will re-review when requested.